contents/blog/startup-ops-toolkit.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2021-03-23
+title: The Startup Ops Toolkit
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Ok, so you’ve read *Zero to One* and *The Hard Thing About Hard Things*, but what are all the boring admin things that are actually going to drain 40% of your time which no one has told you about?
+
+<iframe src="https://giphy.com/embed/pp1H7Z5TX3z44" width="480" height="245" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/reaction-cloudy-with-a-chance-of-meatballs-pp1H7Z5TX3z44">via GIPHY</a></p>
+
+More importantly, what are the tools you can use to streamline and automate these?
+
+**Spoiler alert**: it’s Deel + Gusto + Fondo + Pry + Brex + some free contract templates + Google Sheets/Docs
+
+This article is US-focused, but most of it applies to any country. I won’t offer multiple options here - these are the tools that I have settled on using personally after a bunch of research and trying out different things.
+
+By using these, hopefully I can save you a bunch of time and precious $$$ trying to figure out how to put together contracts, pay people on time, manage your money and comply with laws you are only just finding out exist.
+
+## Paying people
+
+You’re going to be hiring people either as permanent employees or as contractors. You will need the appropriate contract, and a system for automatically paying them.
+
+<iframe src="https://giphy.com/embed/l0G18S6f5RfkI8JNK" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/season-12-the-simpsons-12x14-l0G18S6f5RfkI8JNK">via GIPHY</a></p>
+
+### Permanent employees
+
+I like <a target="_blank" rel="noopener noreferrer nofollow" href="https://gusto.com/">Gusto</a> a lot (US only). They make the process pretty seamless for you and your new hire, offer tantalizing discounts for early stage startups, and integrate things like state tax registrations, healthcare benefits and mandatory insurance quite nicely. Their phone support is great, but avoid email or chat. Gusto charge $39/month plus $6 per employee.
```

Comment:
We need to escape dollar symbols, else it does this:
<img width="910" alt="Screenshot 2021-03-23 at 11 49 24" src="https://user-images.githubusercontent.com/47497682/112141883-d13e2700-8bcd-11eb-8f17-f2da22f7bd81.png">

Reply:
Done 👍